test/unit/math/mix/fun/acosh_test.cpp

```diff
@@ -7,7 +7,7 @@ TEST(mathMixMatFun, acosh) {
     return acosh(x1);
   };
   for (double x : stan::test::internal::common_args())
-    stan::test::expect_unary_vectorized(x);
+    stan::test::expect_unary_vectorized<stan::test::PromoteToComplex::No>(f, x);
```

Comment:
I'm not clear on what this template parameter is doing.  This function should work with `x` promoted to `std::complex`, but I'm guessing the tests won't work that way.

Reply:
This template parameter is turning of the complex tests. Internally could this be because I use the default imaginary value of 0.0? Should these tests use the input value as the imaginary part as well?

Reply:
As long as we test with non-zero imaginary components, it's fine.